transit/tests/api_test/test_api_user.py

```diff
@@ -0,0 +1,93 @@
+from django.contrib.auth.models import User
+from rest_framework import status
+
+from transit.rest_api.urls import UserViewSet
+from transit.tests.api_test import ApiTest
+
+
+class ApiLoginTest(ApiTest):
+    _TEST_NEW_USER_USERNAME = 'newuser'
+    _TEST_NEW_USER_PASSWORD = 'newuserpass'
+
+    def setUp(self):
+        super(ApiLoginTest, self).setUp()
+        self.payload = {
```

Comment:
Why added this setup if there's existing mixin for user login?

Reply:
let me check this. but probably  you are right.